src/_igraph/graphobject.c

```diff
@@ -18628,6 +18768,8 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "  current membership vector any more.\n"
    "@return: the community membership vector.\n"
   },
+  {"community_voronoi", (PyCFunction) igraphmodule_Graph_community_voronoi,
+     METH_VARARGS | METH_KEYWORDS, "Finds communities using Voronoi partitioning"},
```

Comment:
Each function should be listed once in this section. It seems to me that this is duplicated: it's already listed in a more complete form below.